cove/templates/terms.html

```diff
@@ -146,6 +176,37 @@ <h3>Understanding website visitor and traffic patterns</h3>
   {% endif %}
   <p>The data is kept indefinitely, in pseudonymised form.</p>
 
+{% if app == 'cove_ocds' %}
+  <h4>Google analytics</h4>
+
+  <p>Personal data we collect:</p>
+  <ul class="simple">
+    <li>Your IP address</li>
+    <li>Referrer (what page you arrived at one of our web pages from)</li>
+    <li>Information about your device, OS and browser</li>
+  </ul>
+  <p>We do not use this data to personally identify individuals, but it is possible that it could be used to do so, particularly if combined with other datasets.</p>
+  <p>Data processors: Google Analytics, Open Data Services Co-operative Limited</p>
+  <p>The data controller (Open Contracting Partnership) is based in the USA.</p>
+  <p>Data is transferred to Google Analytics, who may transfer data to third countries (non-EEA).</p>
+  <p>The data is kept indefinitely, in pseudonymised form.</p>
+
+  {% if hotjar.id %}
+  <h4>Hotjar</h4>
+
+  <p>In order to understand the behaviour of website visitors, to help us understand the most popular features and understand potential sources of frustration for website users, we are currently running a time-limited experiment using <a href="https://www.hotjar.com/">Hotjar</a> behaviour tracking.</p>
+
+  <p>The experiment runs from {{ hotjar.date_info }}.</p>
+
+  <p>Hotjar is a technology service that helps us better understand our users experience (e.g. how much time they spend on which pages, which links they choose to click, what users do and don’t like, etc.) and this enables us to build and maintain our service with user feedback. Hotjar uses cookies and other technologies to collect data on our users’ behavior and their devices (in particular device's IP address (captured and stored only in anonymized form), device screen size, device type (unique device identifiers), browser information, geographic location (country only), preferred language used to display our website). Hotjar stores this information in a pseudonymized user profile. Neither Hotjar nor we will ever use this information to identify individual users or to match it with further data on an individual user. For further details, please see Hotjar’s privacy policy by clicking on <a href="https://www.hotjar.com/legal/policies/privacy">this link</a>.</p>
```

Comment:
s/our users experience/our users' experience/

Reply:
I also think that the double-parentheses could be avoided with the use of a colon:

Hotjar uses cookies and other technologies to collect data on our users’ behavior and their devices, in particular: device's IP address (captured and stored only in anonymized form),